tests/include/test/drivers/size.h

```diff
@@ -62,16 +62,17 @@
  */
 #define TEST_KEY_CONTEXT_STORE_PUBLIC_KEY   0
 
-/** \def TEST_KEY_CONTEXT_USE_SIZE_FUNCTION
+/** \def TEST_KEY_CONTEXT_SIZE_FUNCTION
  *
- * If this is true a size_function is implemented.
- * If this is false, there is no size_function implemented.
+ * If this is defined a size_function is implemented.
```

Comment:
What about "If this is defined a driver size_function entry point is implemented." ?